### ISSUE TITLE
Incorrect height of the IFrame of the email body from Jira

### ISSUE THREAD

**Issue by Bockser** [nextcloud/mail#8941]
### Steps to reproduce

1. Login to Nextcloud server
2. Go to Mail App
3. Open incoming mail
4. Find and open mail from Jira (Self-Hosted)

### Expected behavior

I'm expecting to see messages

### Actual behavior

The emails from Jira always have a height of 8px, making it impossible to read the email

<img width="1403" alt="273813993-233a8225-fb59-4e5d-82b2-59c5c58b0fd2" src="https://github.com/nextcloud/mail/assets/24253204/a33e4344-39bd-45c8-8778-5d904df4629d">

### Mail app version

3.4.1

### Mailserver or service

Mailcow-dockerized

### Operating system

Ubuntu

### PHP engine version

PHP 8.2

### Web server

Apache (supported)

### Database

PostgreSQL

**Comment by ChristophWurst** [comment]
check conf.php for overwrite.cli.url and similar. are the domain names and the schema (http vs https) correct?

**Comment by Bockser** [comment]
Yes, I checked it right away as I was finding a similar issue
<img width="542" alt="Снимок экрана 2023-10-10 в 13 42 14" src="https://github.com/nextcloud/mail/assets/24253204/11723f1e-8e9d-468a-ac91-d93ebdf7949d">

**Comment by Bockser** [comment]
@ChristophWurst The screenshot shows that the iframe content is loaded via https

**Comment by ChristophWurst** [comment]
There is a script called iframeresizer running inside the iframe. Can you try to find out if it's running?

**Comment by Bockser** [comment]
@ChristophWurst Yes, of course! How can I do that?

**Comment by ChristophWurst** [comment]
I don't know. Inspect the iframe with your browser tools? Check for included scripts, etc

**Comment by Bockser** [comment]
@ChristophWurst No, the script "htmlrespone.js" specified in the head of the iframe does not work because when loading the iframe content the error "Refused to load the image 'cid:jira-generated-image-avatar-cedd8d02-fc26-4f9b-a70d-cd28991698e6' because it violates the following Content Security Policy directive: "img-src 'self' data: blob:"".

Even though the script is minimized, you can still see that the logic of iframeResizer is contained in this very script

**Comment by ChristophWurst** [comment]
The CSP error of the inline image should not influence the execution of the script.

**Comment by Bockser** [comment]
> There is a script called iframeresizer running inside the iframe. Can you try to find out if it's running?

Yes, the script runs and I can debug it,
<img width="898" alt="Снимок экрана 2023-10-13 в 18 09 17" src="https://github.com/nextcloud/mail/assets/24253204/bc3884a6-301f-4448-9c5c-821287339fcb">

**Comment by dyoma-bat** [comment]
I have exactly the same problem with emails from jira, although emails from gitlab are displayed correctly

**Comment by Tail870** [comment]
NextCloud Hub 10 (31.0.5), Mail app 5.1.2 - same issue with Jira's mails.

**Comment by ChristophWurst** [comment]
@aseemann please post a screenshot of an email that you can share publicly and also export the .eml, rename to .eml.txt and attach it as well.
This will allow us to reproduce and compare to what you are seeing.

**Comment by mat-m** [comment]
Reproduced with any HTML email (text email are fine) with Mail 5.7.5 on NC 33.0.1.

Height is constant but 150 px:

<img width="582" height="369" alt="Image" src="https://github.com/user-attachments/assets/b1d46bab-8e35-4287-b5d0-1f501de3b4dd" />

<img width="900" height="353" alt="Image" src="https://github.com/user-attachments/assets/e174c636-957d-4126-a85d-f65daecc0299" />
